### DISCUSSION TITLE
设置菜单功能后有时候显示有问题

### DISCUSSION THREAD

**Discussion by luumod** [tangly1024/NotionNext#991] [Q&A] [answered]
设置了Menu与SubMenu后，打开网站的时候有时候会出现菜单位置错位的情况：
**这是正确的显示方式 ：**
![2 2FU__B{Q~P%IW K{DLLSA](https://user-images.githubusercontent.com/93420580/232695977-92b52f33-cb83-4713-a645-0239e6092cee.png)

**然而有时候打开网页的时候，会出现错误的显示方式：**
![%9RGS@U5AHM@J1ZI$ Q6% Y](https://user-images.githubusercontent.com/93420580/232696047-ac8127a7-bfec-423b-85a1-ecb0a2ac0c4a.png)
然后我多刷新了几下就又好了。

这是个Bug吗，还是我自己的问题，我的Notion笔记里面的配置应该是没有问题的。
![image](https://user-images.githubusercontent.com/93420580/232696506-9f5736fd-475a-4a17-8b84-c4787b94987c.png)

**Comment by 22Goose** [comment]
好像是缓存问题[issue#1009](https://github.com/tangly1024/NotionNext/issues/1009)

**Comment by 22Goose** [comment]
我有个问题问一下
你的“关于作者”那个页面的设置是什么啊？
我在notion中也是设置了Menu和Published，但在网站中点击这个页面时无法查看我在notion中写的内容并爆出404错误
我需要在哪里更改或者是否应该添加单独的页面来显示呢？

**Reply by luumod** [reply, marked as answer]
我的配置如下所示，在`page` 的文件里写入内容。
`Menu`的文件的slug指定`/aboutme` ,内容留空
![image](https://user-images.githubusercontent.com/93420580/234254313-211cbc58-61ff-420a-8c9c-e65d9d812561.png)
![image](https://user-images.githubusercontent.com/93420580/234254349-b7292bf2-cf8b-455b-87e4-400b9bf8c87f.png)

**Reply by happyzhangyyds** [reply]
如果把slug的后缀改成about试试看呢。
